pypi_formula_mappings.json

```
@@ -266,6 +266,9 @@
   "flit": {
     "exclude_packages": ["certifi", "docutils", "flit-core"]
   },
+  "fonttools": {
+    "extra_packages": ["brotli"]
+  },
```

there's a [`woff`](https://github.com/fonttools/fonttools/blob/main/setup.py#L102) extra to add brotli support, can we use that (in case they ever change the dependency / cap the version)?
```suggestion
  "fonttools": "fonttools[woff]",
```